File: Data Structures & Algorithms/maximum-subarray/submission-1.java
class Solution {
    public int maxSubArray(int[] nums) {
        int maxSub = Integer.MIN_VALUE;
        int max = 0;
        for(int value : nums) {
            max += value;
            if(max > maxSub) {
                maxSub = max;
            }
            if(max < 0) {
                max = 0;
            }
        }
        return maxSub;
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number}
     */
    longestConsecutive(nums) {
        let Length;
        let Longest = 0;
        const Setarr = new Set(nums);
        for(let value of Setarr ) {
            if(!Setarr.has(value - 1)) {
                Length = 1;
                while(Setarr.has(value + Length)) {
                    Length++;
                }
                Longest = Math.max(Longest, Length);

            }
        }
        return Longest;
    }
}
